<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="Failed suite [suite of suites]" guice-stage="DEVELOPMENT">
  <listeners>
    <listener class-name="com.fliplearn.flipapp.helper.Base"/>
    <listener class-name="com.fliplearn.flipapp.helper.RetryListener"/>
    <listener class-name="com.fliplearn.flipapp.report.CustomTestNGReporter"/>
  </listeners>
  <test thread-count="5" name="Event(failed)">
    <classes>
      <class name="com.fliplearn.flipapp.testcases.PrimeResources">
        <methods>
          
          <include name="verifyGettingValidationMsgWithIncorrctCoupn"/>
        </methods>
      </class> <!-- com.fliplearn.flipapp.testcases.VMC -->
    </classes>
  </test> <!-- Event(failed) -->
</suite> <!-- Failed suite [suite of suites] -->
